spec/lib/declare_schema/generator_spec.rb

```diff
@@ -6,44 +6,85 @@
   end
 
   it "generates nested models" do
-    Rails::Generators.invoke('declare_schema:model', %w[alpha/beta one:string two:integer])
+    generate_model 'alpha/beta', 'one:string', 'two:integer'
 
-    expect(File.exist?('app/models/alpha/beta.rb')).to be_truthy
-
-    expect(File.read('app/models/alpha/beta.rb')).to eq(<<~EOS)
+    expect_model_definition_to_eq('alpha/beta', <<~EOS)
       class Alpha::Beta < #{active_record_base_class}
-  
+
         fields do
           one :string, limit: 255
           two :integer
         end
-  
+
       end
     EOS
 
-    expect(File.read('app/models/alpha.rb')).to eq(<<~EOS)
+    expect_model_definition_to_eq('alpha', <<~EOS)
       module Alpha
         def self.table_name_prefix
           'alpha_'
         end
       end
     EOS
 
-    expect(File.read('test/models/alpha/beta_test.rb')).to eq(<<~EOS)
+    expect_test_definition_to_eq('alpha/beta', <<~EOS)
       require 'test_helper'
-      
+
       class Alpha::BetaTest < ActiveSupport::TestCase
         # test "the truth" do
         #   assert true
         # end
       end
     EOS
 
-    expect(File.exist?('test/fixtures/alpha/beta.yml')).to be_truthy
+    case Rails::VERSION::MAJOR
```

Comment:
These cases all look identical. Copy/paste bug?

Reply:
There are slight differences. Rails 6 introduces `https` to the link instead of `http`, and Rails 5 corrected some spacing in the formatting of the file.